content/docs/command-reference/exp/remove.md

```diff
@@ -56,3 +57,43 @@ $ dvc exp list
 master:
         exp-e6c97
 ```
+
+The same applies to queued experiments but they won't have a name to give to
+`dvc exp remove` yet unless you specify one. Alternatively, you can use their ID
+instead (shown when queued and by `dvc exp show`).
+
+Let's queue a few experiments in different ways and then delete some of them:
+
+```dvc
+$ dvc exp run --queue -S train.min_split=64
+Queued experiment 'e41d5b4' for future execution.
+$ dvc exp run --queue -S train.min_split=32 --name split32
+Queued experiment '5751540' for future execution.
+$ dvc exp run --queue -S train.min_split=16 --name split16
+Queued experiment '8de9a6c' for future execution.
+
+$ dvc exp e41d5b4 split16
+
+$ dvc exp show --include-params=train.min_split --no-pager
+```
+
+```table
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created      ┃ State  ┃ avg_prec ┃ roc_auc ┃ train.min_split ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace             │ -            │ -      │  0.57553 │ 0.94652 │ 2               │
+│ master                │ Aug 02, 2021 │ -      │  0.53252 │  0.9107 │ 2               │
+│ └── 5751540 [split32] │ 04:57 PM     │ Queued │        - │       - │ 32              │
+└───────────────────────┴──────────────┴────────┴──────────┴─────────┴─────────────────┘
+```
```

Comment:
I think we should use `exp list` instead to simplify this example. `exp show` can be noisy.

Reply:
I tried on my computer that `dvc list` could not show queued exps.

Reply:
Indeed, neither `dvc exp list` nor `dvc exp list --all` shows the queued experiments. It looks the only way to get the list of queued experiments is via `dvc exp show`.

Reply:
OK let's keep the `exp show` table then. Thanks

BTW should `exp list` print queued exps? Is there an issue in the core repo perhaps?

Reply:
TBH I think the best solution is moving all of this queue functionality into a `queue` command as suggested in https://github.com/iterative/dvc/issues/5615. I have avoided it because it's a bigger change, but I think it will be needed soon, especially now that we are working on remote execution.